src/app/components/publiclists/challengelist/challengelist.component.html

```diff
@@ -1,58 +1,93 @@
 <section class="ev-sm-container ev-view challenge-container">
-
+  <mat-tab-group>
+    <mat-tab label="Ongoing Challenges [ {{ ongoingChallenges.length }} ]">
```

Comment:
Please change square brackets to round brackets and also the spacing.
`Ongoing Challenges [ {{ ongoingChallenges.length }} ]` --> `Ongoing Challenges ({{ongoingChallenges.length}})`